### PR TITLE
[DUOS-3014] Replace 'Request a Library Card' button with text guidance

### PR DESCRIPTION
**Pull request by PintoGideon**
### Addresses

https://broadworkbench.atlassian.net/browse/DUOS-3014

### Summary

If the user does not have any library cards, replace the button to request a card with some textual guidance.

----
Have you read [Terra's Contributing Guide](https://github.com/DataBiosphere/terra-ui/wiki/Contributor-Guide) lately? If not, do that first.

- Label PR with a Jira ticket number and include a link to the ticket
- Label PR with a security risk modifier [no, low, medium, high]
- PR describes scope of changes
- Get a minimum of one thumbs worth of review, preferably two if enough team members are available
- Get PO sign-off for all non-trivial UI or workflow changes
- Verify all tests go green
- Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

File: src/pages/user_profile/ResearcherStatus.js [hunk 1]
```diff
@@ -99,15 +99,10 @@ export default function ResearcherStatus(props) {
...
-          style={{
-            marginTop: '10px',
-            marginBottom: '50px'
-          }} >
-          Request a Library Card
-        </button>
+        <p style={{
+          marginTop: '10px',
+          marginBottom:'50px'
+        }}>You must have a Library Card to submit a data access request. To obtain one, your Institutional Signing Official must register in DUOS, request and receive Signing Official permissions, and issue you a Library Card.</p>
```
Comment (rushtong) [inline src/pages/user_profile/ResearcherStatus.js]: This leaves the `goToRequestRole` method unused. 
That takes the user to `/request_lc` which now, no longer needs a route in `Routes.js`.
That takes us to `RequestForm` which now no longer uses the prop `isRequestLCPage`
This is essentially the "Request Library Card" feature so this code change makes that code path unreachable now. 
We should potentially do two things here
1. Confirm with @solideoglori that we no longer have a need for this feature
2. If so, remove all of the unreachable code

Comment (solideoglori) [reply src/pages/user_profile/ResearcherStatus.js]: 1. Yes, we no longer need.
2. Agreed!

**Review by aarohinadkarni — APPROVED** [review]
looks good! 👍

File: src/pages/user_profile/RequestForm.js [hunk 2]
```diff
@@ -44,19 +43,6 @@ export default function SupportRequestsPage(props) {
       extraRequest: undefined
     };
   }
```
Comment (rushtong) [inline src/pages/user_profile/RequestForm.js]: The previous if condition here (`if (props.isRequestRolePage)`) is always true so we don't need that any longer. It was only used to distinguish between the LC request vs Role request.

File: src/Routes.jsx [hunk 3]
```diff
@@ -66,7 +66,6 @@ const Routes = (props) => (
     <Route path="/anvil_dms_policy" component={AnVILDMSPolicyInfo} />
     <AuthenticatedRoute path="/profile" component={UserProfile} props={props} rolesAllowed={[USER_ROLES.all]} />
     <AuthenticatedRoute path="/request_role" component={RequestForm} props={Object.assign({}, props, {isRequestRolePage: true})} rolesAllowed={[USER_ROLES.all]} />
```
Comment (rushtong) [inline src/Routes.jsx]: See related comment in `RequestForm` ... we don't need the `isRequestRolePage` property any more - it is always true so we can simplify both this props assignment and the `RequestForm` component by removing it.

File: src/pages/user_profile/ResearcherStatus.js [hunk 4]
```diff
@@ -10,20 +10,13 @@ export default function ResearcherStatus(props) {
...
   const [hasCard, setHasCard] = useState(true);
 
-  const goToRequestRole = () => {
-    pageProps.history.push({
-      pathname: '/request_lc',
-      state: { data: profile }
-    });
-  };
-
+
```
Comment (rushtong) [inline src/pages/user_profile/ResearcherStatus.js]: There is a stray space here that eslint is reporting on

**Review by rushtong — COMMENTED** [review]
See comments inline, I think we can do a little more cleanup here to keep the logic easier to grok.

**Review by rushtong — APPROVED** [review]
Looks good, thank you 👍🏽

**Pull request merged**